client/js/components/assessments/item.jsx

```diff
@@ -25,9 +25,6 @@ export default class Item extends React.Component{
     checkedResponse   : React.PropTypes.object.isRequired,
 
     selectAnswer      : React.PropTypes.func.isRequired,
-
```

Comment:
This isn't getting used for anything and was complaining about the wrong prop type. I removed it for now. We can put it back in later if we use it.